frontend/src/pulse/components/PulseEditChannels.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component, PropTypes } from "react";
 
 import _ from "underscore";
+import i from "icepick";
```

Comment:
Yes, you can give it whatever alias you like, or do `import { assocIn } from "icepick";`

But with `map` and `for (... of ...)` loops it's become a lot less common to write `for (i = 0;...)` style loops in JS anyway.